core/src/main/kotlin/apiDocs/RunsDocs.kt

```diff
@@ -309,7 +309,7 @@ val getRuleViolationsByRunId: OpenApiRoute.() -> Unit = {
                             limit = 20,
                             offset = 0,
                             totalCount = 1,
-                            sortProperties = listOf(SortProperty("packageUrl", SortDirection.ASCENDING))
+                            sortProperties = listOf(SortProperty("rule", SortDirection.ASCENDING))
```

Comment:
I'm confused. This clearly is not a doc-only change. Shouldn't the changes in this file and the changes in `core/src/main/kotlin/api/RunsRoute.kt` from the previous commit be combined in one commit, @mnonnenmacher?

Reply:
This _is_ a doc-only change; it only affects the examples in Swagger documentation.

Reply:
To me it's fine the way it is.